src/service/AppealSubmissionService.ts

```diff
@@ -0,0 +1,37 @@
+import axios, { AxiosRequestConfig } from 'axios';
+import { Appeal } from 'app/models/Appeal';
+
+export class AppealSubmissionService {
+
+    constructor(private readonly appealsApiUrl: string) {
+        this.appealsApiUrl = appealsApiUrl;
+    }
+
+    public async submitAppeal(appealData: Appeal, companyId: string, token: string): Promise<any> {
```

Comment:
Company number is in `Appeal` object already so I suggest `companyId: string` argument is removed from function declaration. It make me confused and it opens API for misuse allowing API call that passes company A in URI and company B in payload.

Reply:
Good point, make's sense 👍 I was thinking of verifying the company id passed in the URI and the one in the payload is equal on the API side